File: servidor/routes/comentarios.go
package routes

import (
	/*"proyectos/proyecto-leo/servidor/middleware"*/
	"proyectos/proyecto-leo/servidor/services"

	"github.com/gin-gonic/gin"
)

func ComentariosRoutes(route *gin.Engine) {

	userGroup := route.Group("/comentarios")
	{
		userGroup.GET("/find", services.ListarComentarios)
		userGroup.GET("/find/:id", services.ListarComentariosPorPlato)
		userGroup.DELETE("/delete/:id", services.EliminarComentario)
		userGroup.PUT("/update/:id", services.ActualizarComentario)
		userGroup.POST("/add", services.AgregarComentario)
	}
}

File: servidor-express/src/routes/comentarios.js
const router = require('express').Router()
const comentariosController = require('../controllers/comentariosController')

router.get("/find", comentariosController.ListarComentarios)
router.get("/find/:id", comentariosController.ListarComentariosPorPlato)
router.delete("/delete/:id", comentariosController.EliminarComentario)
router.put("/update/:id", comentariosController.ActualizarComentario)
router.post("/add", comentariosController.AgregarComentario)

module.exports = router